tools/release-init.sh

```diff
@@ -44,10 +44,14 @@ rm "${project_dir}/dependencies.yml.bak" || exit 1
 sed -i.bak -e "s/^let versionStr =.*/let versionStr = \"${realm_version}\"/" "${project_dir}/Package.swift"
 rm "${project_dir}/Package.swift.bak" || exit 1
 
+# update CHANGELOG.md
 RELEASE_HEADER="# $realm_version Release notes" || exit 1
 sed -i.bak -e "1s/.*/$RELEASE_HEADER/" "${project_dir}/CHANGELOG.md" || exit 1
+sed -i.bak -e "/.*\[#????\](https.*/d" "${project_dir}/CHANGELOG.md"
```

Comment:
This removes any remaining template lines from CHANGELOG.md if nobody else removed them. 
Either of these two, by matching on `[#????](https`
`* <New feature description> (PR [#????](https://github.com/realm/realm-core/pull/????))`
`* <How do the end-user experience this issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)`

Reply:
💯